Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [package]
 name = "lsr"
 version = "0.1.0"
+edition = "2021"
```

Comment:
Use latest edition (without this line `extern crate` declarations are needed to compile).